docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -191,6 +191,8 @@ Adaptive Batch Scheduler will only decide parallelism for operators whose parall
 
 - **Batch jobs only**: Adaptive Batch Scheduler only supports batch jobs. Exception will be thrown if a streaming job is submitted.
 - **ALL-EXCHANGES-BLOCKING jobs only**: At the moment, Adaptive Batch Scheduler only supports jobs whose [shuffle mode]({{< ref "docs/deployment/config" >}}#execution-batch-shuffle-mode) is `ALL-EXCHANGES-BLOCKING`.
+- **The decided parallelism can only be a power of 2**: In order to make the subpartitoins evenly consumed by downstream tasks, user should configure the [`jobmanager.adaptive-batch-scheduler.max-parallelism`]({{< ref "docs/deployment/config" >}}#jobmanager-adaptive-batch-scheduler-max-parallelism) to be a power of 2 (2^N), and the decided parallelism will also be a power of 2 (2^M and M < N).
```

Comment:
subpartitoins -> subpartitions

Reply:
> In order to make the subpartitoins evenly consumed by downstream tasks

maybe "In order to ensure downstream tasks to consume the same count of subpartitions"?

Reply:
> user should configure the XXX ...

maybe "the configuration XXX should be set to be a power of 2"

Reply:
`M < N` -> `M <= N`